tests/integration/__init__.py

```diff
@@ -262,7 +262,8 @@ def execute_until_pass(session, query):
     while tries < 100:
         try:
             return session.execute(query)
-        except ConfigurationException:
+        except (ConfigurationException, AlreadyExists):
+            log.warn("Recieved already exists from query {0}   not exiting".format(query))
```

Comment:
"already exists" is not completely accurate. You would be handling "non existing" if it's ConfigurationException on a DROP